submissions/dafen173/js-dom/style.css

```diff
@@ -0,0 +1,163 @@
+
+
+.maincontainer {
+    width: 100%;
+    display: flex;
+    flex-wrap: wrap;
+}
+
+
+
+.mainmenubtn {
+    background-color: #696969;
+    color: white;
+    cursor: pointer;
+    padding:20px;
+    
+}
+
+/*
+.mainmenubtn:hover {
+    background-color: green;
+    }
+*/   
+
+.dropdown {
+    /*position: relative;*/
+    /*display: inline-block;*/
+    float: left;
```

Comment:
This is an old school way for defining blocks position. If browser support allows better to use flex or grid.